liquibase-core/src/main/java/liquibase/change/AbstractChange.java

```diff
@@ -411,6 +415,57 @@ public CheckSum generateCheckSum() {
         return CheckSum.compute(new StringChangeLogSerializer().serialize(this, false));
     }
 
+    /**
+     * Gets sql content from provided file
+     *
+     * @param path                      path to sql file
+     * @param encoding                  The encoding of the file containing SQL statements
+     * @param isRelativeToChangelogFile is file path relative to changelog
+     * @return sql content
+     * @throws UnexpectedLiquibaseException in case of problems with reading file
+     */
+    protected String getChangeContent(String path, String encoding, Boolean isRelativeToChangelogFile) {
```

Comment:
The getChangeContent() and openSqlStream() method only really applies to a couple subclasses of AbstractChange so I don't think we should add these methods to AbstractChange. I like the consolidation of logic, but since the methods don't apply to all AbstractChange implementations we shouldn't put it there.

We could make a new AbstractSqlChange subclass that has just these methods, but I think it would be better to just add getChangeContent() to StreamUtil like openSqlStream() is. That way we can still consolidate all the logic, but we don't end up with a method on AbstractChange that doesn't apply to most implementations.

Am I missing something useful with the methods on AbstractChange?